crates/orderbook/src/main.rs

```diff
@@ -413,6 +420,30 @@ async fn main() {
                     "ZeroEx",
                     metrics.clone(),
                 )),
+                PriceEstimatorType::Quasimodo => Box::new(InstrumentedPriceEstimator::new(
+                    QuasimodoPriceEstimator {
+                        api: Arc::new(HttpSolverApi {
+                            name: "quasimodo-price-estimator",
+                            network_name: network_name.to_string(),
+                            chain_id,
+                            base: args.quasimodo_solver_url.clone(),
+                            client: client.clone(),
+                            config: SolverConfig {
+                                api_key: None,
+                                max_nr_exec_orders: 100,
```

Comment:
I think it makes no difference, but we could set the number to 1

Reply:
I'm thinking if this setting should be per request?

Reply:
Since we only have 1 order (the order we are using to quote a price estimate), we can use `1` here.

THB I don't think it makes much of a difference since we have less than the maximum.

Reply:
Yes, it makes no difference.